### PR TITLE
Added change active sheet in the How to section

### PR DESCRIPTION
**Pull request by babu-periyasamy**
Description:

Added the change active sheet md file in How to section in Typescript UG documentation.

### CODE REVIEW

File: ej2-javascript/spreadsheet/ej2-typescript-spreadsheet-toc.html [hunk 1]
```diff
@@ -39,6 +39,7 @@
 <li><a href="/ej2-typescript/spreadsheet/how-to/sort-a-range-by-custom-list">Sort a range by custom list</a></li>
 <li><a href="/ej2-typescript/spreadsheet/how-to/create-a-object-structure">Create a JSON structure</a></li>
 <li><a href="/ej2-typescript/spreadsheet/how-to/print">Print the single multiple sheets</a></li>
+<li><a href="/ej2-typescript/spreadsheet/how-to/change-active-sheet">Change the active sheet in the imported Excel file</a></li>
```
Comment (harivenkateshe) [inline ej2-javascript/spreadsheet/ej2-typescript-spreadsheet-toc.html:42]: Change title like - "Changing the active sheet during importing file"

File: ej2-javascript/spreadsheet/ej2-javascript-spreadsheet-toc.html [hunk 2]
```diff
@@ -38,6 +38,7 @@
 <li><a href="/ej2-javascript/spreadsheet/how-to/sort-a-range-by-custom-list">Sort a range by custom list</a></li>
 <li><a href="/ej2-javascript/spreadsheet/how-to/create-a-object-structure">Create a JSON structure</a></li>
 <li><a href="/ej2-javascript/spreadsheet/how-to/print">Print the single multiple sheets</a></li>
+<li><a href="/ej2-javascript/spreadsheet/how-to/change-active-sheet">Change the active sheet in the imported Excel file</a></li>
```
Comment (harivenkateshe) [inline ej2-javascript/spreadsheet/ej2-javascript-spreadsheet-toc.html:41]: Change title like - "Changing the active sheet during importing file"

File: ej2-javascript/spreadsheet/how-to/change-active-sheet.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+title: Set an imported excel file's active sheet in ##Platform_Name## Spreadsheet control | Syncfusion
+description: Learn here all about set an imported excel file's active sheet in Syncfusion ##Platform_Name## Spreadsheet control of Syncfusion Essential JS 2 and more.
+platform: ej2-javascript
+control: Change the active sheet in the imported Excel file 
+publishingplatform: ##Platform_Name##
+documentation: ug
+domainurl: ##DomainURL##
+---
+
+## Set an imported excel file's active sheet in ##Platform_Name## Spreadsheet control
```
Comment (harivenkateshe) [inline ej2-javascript/spreadsheet/how-to/change-active-sheet.md:12]: Changing the active sheet during importing file in the ##Platform_Name## Spreadsheet control

File: ej2-javascript/spreadsheet/how-to/change-active-sheet.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+---
```
Comment (harivenkateshe) [inline ej2-javascript/spreadsheet/how-to/change-active-sheet.md:1]: layout: post
title: Changing the active sheet while importing a file in ##Platform_Name## Spreadsheet control | Syncfusion
description: Learn here all about changing the active sheet index while importing a file in Syncfusion ##Platform_Name## Spreadsheet control of Syncfusion Essential JS 2 and more.
platform: ej2-javascript
control: Spreadsheet
publishingplatform: ##Platform_Name##
documentation: ug
domainurl: ##DomainURL##

**Review by gobalakrishnans — APPROVED** [review]

**Pull request merged**